packages/peregrine/lib/talons/Gallery/__tests__/useGalleryItem.spec.js

```diff
@@ -0,0 +1,8 @@
+import { useGalleryItem } from '../useGalleryItem';
+test('returns object handelLinkClick as a function', () => {
+    const expectedReturnValue = {
+        handleLinkClick: () => {}
+    };
+    const props = useGalleryItem({});
+    expect(JSON.stringify(props)).toBe(JSON.stringify(expectedReturnValue));
```

Comment:
Since this test is specifically looking for `handleLinkClick` and wants it to be a `function`, you could use [objectContaining](https://jestjs.io/docs/expect#expectobjectcontainingobject) combined with [expect.any](https://jestjs.io/docs/expect#expectobjectcontainingobject) for this.

```js
expect(props).toEqual(
  expect.objectContaining({
    handleLinkClick: expect.any(Function)
  })
)
```

Doesn't block a 👍 , just more of an fyi :)

Reply:
Ah ha! i was struggling for some way other than `JSON.stringify` to verify `handleLinkClick` was a function. Thanks :)

Reply:
Change has been made.